roles/installer/README.md

```diff
@@ -31,6 +31,7 @@ Run the installer with the `satellite` scenario, setting the initial organizatio
     - role: redhat.satellite_operations.installer
       vars:
         satellite_installer_scenario: satellite
+        satellite_installer_package: satellite-installer-satellite
```

Comment:
this looks wrong and thanks to the branding in the makefile we already have the correct default:

https://github.com/RedHatSatellite/satellite-operations-collection/blob/b52ec23bbfe97a44bf4f142f6783a59e9db00bfb/roles/installer/defaults/main.yml#L3